template/app/build.gradle

```diff
@@ -57,22 +57,22 @@ android {
 dependencies {
     //Kotlin
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.1"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.2"
 
     // Android
-    implementation 'androidx.core:core-ktx:1.7.0-beta02'
-    implementation 'androidx.appcompat:appcompat:1.3.1'
-    implementation 'androidx.coordinatorlayout:coordinatorlayout:1.1.0'
-    implementation 'androidx.preference:preference-ktx:1.1.1'
-    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1'
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.3.1'
+    implementation 'androidx.core:core-ktx:1.9.0-alpha01'
```

Comment:
we no longer need to keep up with unstable branches, this one was because of edge to edge that is now supported.